.travis.yml

```diff
@@ -3,23 +3,58 @@ node_js:
   - lts/*
 git:
   depth: 3
+addons:
+  apt:
+    packages:
+      # Needed hack to have colored output, see: travis-ci/travis-ci/#7967
+      # This intstalls unbuffer which reenables colored output for builds
+      # with secret environment variables
+      - expect-dev
 cache:
   - pip
   - npm
-install:
-  - pip install grow==0.7.4 --user
-  - grow --version
-  - which grow
-  - npm ci
-before_deploy:
-  - npm run build:staging
-deploy:
-  provider: gae
-  keyfile: ".staging.key.json"
-  project: amp-dev-staging
-  skip_cleanup: true
-  on:
-    branch: future
 before_install:
 - openssl aes-256-cbc -K $encrypted_d4a09416a845_key -iv $encrypted_d4a09416a845_iv
   -in .staging.key.json.enc -out .staging.key.json -d
+- gcloud auth activate-service-account --key-file=.staging.key.json
```

Comment:
curios: why is this step needed now?

Reply:
Had auth errors for the `gsutil cp ...` calls otherwise.